api/core/project/v1/project.proto

```diff
@@ -43,3 +44,14 @@ message PagerDuty {
   // https://developer.pagerduty.com/api-reference/docs/CONCEPTS.md#services
   repeated string service_ids = 1;
 }
+
+message Link {
+  string name = 1;
+  string url = 2;
+  string image_path = 3; // Can be an s3 location, path on the server, etc.
+}
+
+message LinkEntity {
+  string name = 1;
+  repeated Link links = 2;
```

Comment:
I got the idea from Scarlett's doc. It would go something like this:

A `LinkEntity` could be Kibana, Grafana, Deploy system, etc. Inside each one, there could be multiple links, i.e. for staging, production, or in the case of Kibana, different queries for different use cases of looking at logs.

LinkEntity 1
```
{
  name: Kibana
  links: [
{
  name: Kibana Staging,
  url: “kibana-staing.lyft.net”, 
  asset_dir: “/frontend/…”
},
{
  name: Kibana Production, 
  url: “kibana.lyft.net”,
  asset_dir: “/frontend/…”
}
  ]
}
```

LinkEntity 2
```
{
  name: Grafana
  links: [
{
  name: Grafana Staging,
  url: “grafana-staging.lyft.net”, 
  asset_dir: “/frontend/…”
},
{
  name: Grafana Production, 
  url: grafana.lyft.net”,
  asset_dir: “/frontend/…”
}
  ]
}
```

Inside Projects:
```
links = {
Kibana,
Grafana,
etc.
}
```